Documentation/dev/On-disk-format.rst

```diff
@@ -1045,11 +1051,12 @@ Off   Size Type  Description
 0x30
 ===== ==== ===== =======================
 
+.. _DEV_ITEM_CONTENT:
 
-DEV_ITEM (d8)
-^^^^^^^^^^^^^
+DEV_ITEM (0xd8)
+^^^^^^^^^^^^^^^
 
-(1, device id) TODO
+Key format: (1,0xd8,device id)
```

Comment:
For the key format part, please avoid use immediate numbers, and use the `(DEV_ITEMS, DEV_ITEM, <device id>)`, just like what we did in print-tree.

Reply:
I'll change it at merge time, but if you hit similar situations, feel free to change them all.